models/favourite.models.js

```diff
@@ -64,7 +61,40 @@ async function addFavourite(user_id, newFavourite) {
   }
 }
 
+async function removeFavourite(favourite_id) {
+  try {
+    if (!parseInt(favourite_id)) {
+      throw { 
+          status: 400,
+          msg: `You passed ${favourite_id}. Favourite id should be a number.` 
+        };
```

Comment:
should we make a util for check if NaN, with generic error ${param} is not a number?